tests/expectations/compiler/symbols/illegal_names_fail.out

```diff
@@ -1,34 +1,24 @@
-Error [ETYC0372132]: `fooaleobar` is an invalid program name. A program cannot have "aleo" in its name.
+Error [ENV03711001]: `fooaleobar` is an invalid program name. A program cannot have "aleo" in its name.
     --> compiler-test:5:9
      |
    5 | program fooaleobar.aleo {
      |         ^^^^^^^^^^
-Error [ETYC0372133]: Record name `FooBar` is prefixed by the record name `Foo`. Record names must not be prefixes of other record names.
-    --> compiler-test:17:12
-     |
-  17 |     record FooBar {
-     |            ^^^^^^
-Error [ETYC0372133]: Record name `FooBarBaz` is prefixed by the record name `FooBar`. Record names must not be prefixes of other record names.
-    --> compiler-test:12:12
-     |
-  12 |     record FooBarBaz {
-     |            ^^^^^^^^^
```

Comment:
This error seems to have gone away. It tests a very specific (and weird) pattern that apparently is an attack vector. We should probably bring it back. (I believe this is only for records)